atlp-rwanda/trojans-ec-fe
# 184760049 Get users and assign role

### **What does This PR do? **
Get users and assign role.

### **Description of the task completed**
This branch contains react code for getting users data when you are admin otherwise you cannot be able to get all that users after getting them you can also assign role them either admin, seller or buyer.

### **How should it be tested manually**
in order to test is you required to login as admin then after login click button for users the website will direct you to page of all users when you react there you are able to see all users and also assign role to them. [PR Preview](https://trojans-ec-fe-git-ft-recommended-products-184760065-atlp-trojan.vercel.app/login)

### **Any background context you want to provide ?**
*N/A**

### **What are the relevant pivotal tracker stories**
https://www.pivotaltracker.com/n/projects/2628461/stories/184760049

### **Screenshots(if appropriate)**
<img width="846" alt="get users" src="https://user-images.githubusercontent.com/112135715/235367328-7027b8d5-ff2d-4b30-bb92-a1e25b81167f.png">
<img width="870" alt="assign role" src="https://user-images.githubusercontent.com/112135715/235367341-db2283a1-ffc5-46ac-a11a-a7f19e8541b2.png">
<img width="869" alt="assign role complete" src="https://user-images.githubusercontent.com/112135715/235367368-b8d8984d-f06c-4ffc-805c-d21355493b07.png">
